packages/experimental/Shadow/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@fluentui-react-native/experimental-shadow",
+  "version": "0.0.1",
```

Comment:
Oh are we supposed to start packages at 0.0.1? Whoops.

Reply:
We've historically done 0.1.0. Is there a package we have that is 0.0.1?

Reply:
I don't think its a big deal either way, but beach ball with bump it to 0.1.0 after this gets published anyway since I described this a minor change not a patch.

Reply:
I think I did 0.0.1 simply because that was the default for npm init